docs/clusterinterceptors.md

```diff
@@ -1,52 +1,39 @@
 <!--
 ---
-linkTitle: "ClusterInterceptor"
-weight: 9
+linkTitle: "ClusterInterceptors"
+weight: 5
 ---
 -->
-# ClusterInterceptor
+# `ClusterInterceptors`
 
-A `ClusterInterceptor` is cluster scoped resource that registers a new Interceptor that 
-can be invoked during the processing of a trigger to modify the behavior or payload of Triggers. The
-custom resource describes how an EventListener can connect to a workload that
-is running the interceptor business logic (and in the future what extra
-paramters the interceptor accepts).
+Tekton Triggers ships with the `ClusterInterceptor` Custom Resource Definition (CRD), which allows you to implement a custom cluster-scoped Webhook-style `Interceptor`.
 
-A `ClusterInterceptor` will consist of a custom resource of type ClusterInterceptor that defines it and a 
-deployment/service combination that runs a HTTP server containing the actual interceptor processing logic. The 
-interceptor can process the incoming request, decide to stop or continue processing, and add extra fields that are 
-available to `TriggerBindings` and other interceptors in a chain under the `extensions` field.
+A `ClusterInterceptor` works in tandem with an external Kubernetes v1 Service running custom business logic that receives the event payload from the
```

Comment:
what do you think of "A `ClusterInterceptor` describes or declares an Kubernetes service running custom business logic ....

Reply:
Changed it to "specifies" which to me means both "defines" and "provides/establishes."